File: check_key.js
const fs = require('fs');
const lines = fs.readFileSync('c:/Users/12494/WorkBuddy/20260331205655/client/src/views/admin/UsersMerchant.vue', 'utf8').split('\n');
for(let i=94;i<99;i++) console.log((i+1)+': '+JSON.stringify(lines[i]));

File: check_key.py
import sys
with open(r'c:\Users\12494\WorkBuddy\20260331205655\client\src\views\admin\UsersMerchant.vue', 'r', encoding='utf-8') as f:
    lines = f.readlines()
for i, line in enumerate(lines[94:99], start=95):
    print(f'{i}: {repr(line)}')
